python/ray/tests/conftest.py

```diff
@@ -649,6 +650,9 @@ def call_ray_stop_only():
 def start_cluster(ray_start_cluster_enabled, request):
     assert request.param in {"ray_client", "no_ray_client"}
     use_ray_client: bool = request.param == "ray_client"
+    if os.environ.get("RAY_MINIMAL") == "1" and use_ray_client:
```

Comment:
if we add this for individual tests, there are a lot of them? Also, is it possible to add this to the def like the comment below>? 

```
@pytest.skipif(...)
def start_cluster(..)
```